app/static/js/subscribe/subscribe.js

```diff
@@ -0,0 +1,46 @@
+$(function () {
+    var userFirst = $("#user-first-name-field");
+    var userLast = $("#user-last-name-field");
+    var userEmail = $("#user-email-field");
+    var userPhone = $("#user-phone-field");
+
+    // Nav active state change
+    $(".nav li").removeClass("active");
+    $('a[href=".' + this.location.pathname + '"]').parents('li,ul').addClass('active');
+
+    // Prevents user from typing non numeric keys
+    $("#user-phone-field").keypress(function (key) {
```

Comment:
Instead of using the jquery selector for the field, you can use the variable userPhone as you have initialized it in line 5.